components/apimgt/org.wso2.carbon.apimgt.rest.api.endpoint.registry/src/test/java/org.wso2.carbon.apimgt.rest.api.endpoint.registry/impl/RegistriesApiServiceImplTest.java

```diff
@@ -914,26 +685,18 @@ public void getAllEntriesInRegistry() throws APIManagementException {
 
         List<EndpointRegistryEntry> endpointRegistryEntryList = new ArrayList<>();
 
-        EndpointRegistryEntry endpointRegistryEntry1 = new EndpointRegistryEntry();
-        endpointRegistryEntry1.setEntryId("entry1");
-        endpointRegistryEntry1.setName("Entry Name 1");
-        endpointRegistryEntry1.setMetaData("{mutualTLS: true}");
-        endpointRegistryEntry1.setProductionServiceURL("https://xyz.com");
-        endpointRegistryEntry1.setServiceType(RegistryEntryDTO.ServiceTypeEnum.REST.toString());
-        endpointRegistryEntry1.setServiceCategory(RegistryEntryDTO.ServiceCategoryEnum.UTILITY.toString());
-        endpointRegistryEntry1.setDefinitionURL("https://petstore.swagger.io/v2/swagger.json");
-        endpointRegistryEntry1.setDefinitionType(RegistryEntryDTO.DefinitionTypeEnum.OAS.toString());
+        EndpointRegistryEntry endpointRegistryEntry1 = createRegistryEntry("entry1", "Entry Name 1",
+                "{mutualTLS: true}", "https://xyz.com",
+                RegistryEntryDTO.ServiceTypeEnum.REST, RegistryEntryDTO.ServiceCategoryEnum.UTILITY,
+                "https://petstore.swagger.io/v2/swagger.json", RegistryEntryDTO.DefinitionTypeEnum.OAS,
+                null);
         endpointRegistryEntryList.add(endpointRegistryEntry1);
 
-        EndpointRegistryEntry endpointRegistryEntry2 = new EndpointRegistryEntry();
-        endpointRegistryEntry2.setEntryId("entry2");
-        endpointRegistryEntry2.setName("Entry Name 2");
-        endpointRegistryEntry2.setMetaData("{mutualTLS: true}");
-        endpointRegistryEntry2.setProductionServiceURL("https://xyz2.com");
-        endpointRegistryEntry2.setServiceType(RegistryEntryDTO.ServiceTypeEnum.SOAP_1_1.toString());
-        endpointRegistryEntry2.setServiceCategory(RegistryEntryDTO.ServiceCategoryEnum.DOMAIN.toString());
-        endpointRegistryEntry2.setDefinitionURL("https://petstore.swagger.io/v2/swagger2.json");
-        endpointRegistryEntry2.setDefinitionType(RegistryEntryDTO.DefinitionTypeEnum.GQL_SDL.toString());
+        EndpointRegistryEntry endpointRegistryEntry2 = createRegistryEntry("entry2", "Entry Name 2",
+                "{mutualTLS: false}", "https://xyz2.com",
```

Comment:
Shall we declare variables for "Entry Name 2", "{mutualTLS: false}" and "https://xyz2.com"?
It has been used in many places.

Reply:
Removed the file in https://github.com/wso2/carbon-apimgt/pull/8615